common-scripts/basic-gluster.sh

```diff
@@ -30,6 +30,7 @@ set -x
 
 # enable repositories
 yum -y install centos-release-gluster yum-utils
+yum --enablerepo=centos-gluster*-test install glusterfs-api-devel
```

Comment:
Please do this further below at
~~~
# install NFS-Ganesha build dependencies
~~~

Reply:
Done. Please check the same.